src/main/java/ui/components/FilterTextField.java

```diff
@@ -245,21 +254,29 @@ public void setFilterText(String text) {
     }
 
     /**
-     * Sets the 'cancel' callback, which will be called with Esc is pressed and there is
+     * Sets the 'onCancel' callback, which will be called with Esc is pressed and there is
      * no edit to revert.
      */
-    public FilterTextField setOnCancel(Runnable cancel) {
-        this.cancel = cancel;
+    public FilterTextField setOnCancel(Runnable onCancel) {
+        this.onCancel = onCancel;
         return this;
     }
 
     /**
-     * Sets the 'confirm' callback, which will be called when Enter is pressed, or when the
+     * Sets the 'onConfirm' callback, which will be called when Enter is pressed, or when the
      * contents of the field are manually set with {@link #setFilterText}. The callback will
      * be passed the current contents of the field.
      */
-    public FilterTextField setOnConfirm(Function<String, String> confirm) {
-        this.confirm = confirm;
+    public FilterTextField setOnConfirm(Function<String, String> onConfirm) {
+        this.onConfirm = onConfirm;
+        return this;
+    }
+    
+    /**
+     * Sets the 'onHelp' callback, which will be called when F1 is pressed.
```

Comment:
This method is about binding `Help` to it's implementation, not binding `F1` to help, right?
Another way to look at it is, if we change the Help short cut to `F2` at some point, it should not require a change to this code, and if the code doesn't change, the comment shouldn't change either.
So, is there a better place to explain `F1` is bound to `Help`?